TreeList: update Column Chooser and Header Filter demos (#2571)
DevExpress/devextreme-demos: JSDemos/Demos/TreeList/ColumnChooser/description.md

* *"select"*         

> Users select and deselect checkboxes with column names.

Reviewer: ```suggestion
Users select and deselect check boxes with column names.
```

In *"select"* mode, you can choose whether parent element selection affects child/nested elements. Use the **selection**.[recursive](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/selection/#recursive) property.

> If the column chooser contains a lot of hidden column, you can enable column search UI. Assign **true** to the **search**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/search/#enabled) property.

Reviewer: ```suggestion
If the column chooser contains many hidden columns, you can enable the column search UI. Assign **true** to the **search**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/search/#enabled) property.
```

> In this demo, use checkboxes below the TreeList to toggle the selection and search features.

Reviewer: ```suggestion
In this demo, use the check boxes below the TreeList to toggle the selection and search features.
```

* *"select"*         
Users select and deselect checkboxes with column names.

> The column chooser selection is non-recursive. When a user selects a parent item, nested items remain unselected. To select items recursively, set the **selection**.[recursive](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/selection/#recursive) property to **true**. In this demo, you can use the checkbox under the TreeList component to toggle this property.

Reviewer 2: I'm making changes consistent with the DataGrid demos.

```suggestion
In *"select"* mode, you can choose whether parent element selection affects child/nested elements. Use the **selection**.[recursive](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/selection/#recursive) property.
```

> If the **search**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/search/#enabled) property is set to **true**, users can search through column names.  In this demo, use the checkbox below the TreeList to toggle this feature.

Reviewer 2: ```suggestion
If the column chooser contains a lot of hidden column, you can enable column search UI. Assign **true** to the **search**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnChooser/search/#enabled) property.

In this demo, use checkboxes below the TreeList to toggle the selection and search features.
```
